Source/WebCore/rendering/mathml/RenderMathMLUnderOver.cpp

```diff
@@ -296,10 +296,7 @@ void RenderMathMLUnderOver::layoutBlock(bool relayoutChildren, LayoutUnit pageLo
 {
     ASSERT(needsLayout());
 
-    for (auto& box : childrenOfType<RenderBox>(*this)) {
-        if (box.isOutOfFlowPositioned())
-            box.containingBlock()->insertPositionedObject(box);
-    }
+    handlePositionedAndFloatingChildren();
```

Comment:
ditto, this should probably be after the "!relayoutChildren && simplifiedLayout()" early return.